tests/test_meshing_utilities.py

```diff
@@ -1081,64 +1081,113 @@ def test_meshing_utilities(new_meshing_session):
         [163, 91, 19]
     )
 
-    # Commented due to print only output
-
-    # assert meshing_session.meshing_utilities.get_shared_boundary_face_zones_for_given_cell_zones(cell_zone_id_list=[87]) == "2001 faces on symmetry-xyplane refer to cell zone elbow-fluid \
-    #         55 faces on hot-inlet refer to cell zone elbow-fluid \
-    #         152 faces on cold-inlet refer to cell zone elbow-fluid \
-    #         155 faces on outlet refer to cell zone elbow-fluid \
-    #         268 faces on wall-inlet refer to cell zone elbow-fluid \
-    #         2168 faces on wall-elbow refer to cell zone elbow-fluid"
+    assert not meshing_session.meshing_utilities.get_shared_boundary_face_zones_for_given_cell_zones(
```

Comment:
The testing shows the improvements nicely. Good work!

Reply:
Thank you.